### PR TITLE
Add documentation for max headroom right-sizing

### PR DESCRIPTION
**Pull request by michaelmdresser**

### CODE REVIEW

File: apis.md [hunk 1]
```diff
@@ -28,5 +28,8 @@ Assets API retrieves the backing cost data broken down by individual assets, e.g
 
 __[/savings](https://docs.google.com/document/d/1h_LQuTdwzbzcSonZ49w0lktaaHsulNsIj-O0OZR12fc/edit)__
 
-Exposes a set of APIs for cost optimization insights, e.g. pod-rightsizing and more. 
+Exposes a set of APIs for cost optimization insights.
 
+__[/savings/requestSizing](./api-request-right-sizing.md)
```
Comment (dwbrown2) [inline apis.md:33]: do you like this in its own separate doc vs having all in one Savings API page?

potentially, related... feels like its an opportunity to move that Google doc to this repo...

File: api-abandoned-workloads.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+```
+http://<kubecost-address>/model/savings/abandonedWorkloads
+```
+
+## Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| `days` | int | Number of historical days over which network traffic should be measured. |
+| `threshold` | int | The threshold of traffic at which a workload is determined abandoned. |
```
Comment (michaelmdresser) [inline api-abandoned-workloads.md:13]: @mbolt35 Do you know what the units are here?

File: apis.md (hunk 1 above)
Comment (michaelmdresser) [reply apis.md]: Moved, let me know what you think.

File: api-abandoned-workloads.md (hunk 2 above)
Comment (nikovacevic) [reply api-abandoned-workloads.md:13]: bytes/second

File: api-request-right-sizing.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+```
+A CronJob that spins up 1 pod with 1 container on a cadence faster than it takes the
+job to run.
+3 | ------
+5 |    ------
+7 |        ------
+Correct max for the CronJob: 7+5 = 12
+```
```
Comment (nikovacevic) [inline api-request-right-sizing.md:56]: I don't feel super strongly about it, but I think all of this is implementation detail that doesn't necessarily need to be included here.

**Review by nikovacevic — APPROVED** [review]
Seems good to me, but happy to defer to @dwbrown2 on some specifics

File: api-abandoned-workloads.md (hunk 2 above)
Comment (michaelmdresser) [reply api-abandoned-workloads.md]: Thanks!

**Pull request merged**